lib/hyrax/specs/shared_specs/indexers.rb

```diff
@@ -79,9 +91,10 @@
   end
 
   describe '#to_solr' do
+    before { attributes.each { |k, v| the_resource.set_value(k, v) } }
     it 'indexes basic metadata' do
       expect(indexer.to_solr)
-        .to include(keyword_sim: a_collection_containing_exactly(*attributes[:keyword]),
+        .to include(keyword_sim:   a_collection_containing_exactly(*attributes[:keyword]),
```

Comment:
This change just makes the a_collection... align with a_collection... in the following 2 lines.  No real code change.